src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -118,9 +124,53 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         this.exercise.participationStatus = participationStatus(this.exercise);
         this.isAfterAssessmentDueDate = !this.exercise.assessmentDueDate || moment().isAfter(this.exercise.assessmentDueDate);
         this.exerciseCategories = this.exerciseService.convertExerciseCategoriesFromServer(this.exercise);
+        if (this.exercise.type.toString() === 'programming') {
+            this.getProgrammingExercise();
+        }
         this.subscribeForNewResults();
     }
 
+    /**
+     * The button should be only visible in the dev environment
+     */
+    isInProduction(): boolean {
```

Comment:
This could be a getter if written as `get isInProduction(): boolean {` which means that `()` can be omitted when calling it (thus treating it like a computed property).